### PR TITLE
feat: replace create with prepare

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
## Description

This PR replaces create command functionality with prepare command functionality.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly. For updating Oclif commands documentation use [oclif-dev](https://github.com/oclif/dev-cli#oclif-dev-readme).
- [ ] I have read the **CONTRIBUTION_GUIDE** document.
- [ ] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/commands/create/map.ts [hunk 1]
```diff
@@ -8,13 +8,13 @@ import type { Flags } from '../../common/command.abstract';
...
 import { ProfileId } from '../../common/profile';
-import { prepareMap } from '../../logic/prepare/map';
+import { createMap } from '../../logic/create/map';
 
 export class Map extends Command {
   public static strict = true;
 
   public static description =
-    'Prepares map, based on profile and provider on a local filesystem. Created file contains prepared structure with information from profile and provider files. Before running this command you should have prepared profile (run sf prepare:profile) and provider (run sf prepare:provider)';
+    'Prepares map, based on profile and provider on a local filesystem. Created file contains created structure with information from profile and provider files. Before running this command you should have prepared profile (run sf prepare:profile) and provider (run sf prepare:provider)';
```
Comment (kysely) [inline src/commands/create/map.ts:17]: > (...)  Created file contains created (...)

😄 

```suggestion
    'Creates map, based on profile and provider on a local filesystem. Created file contains structure with information from profile and provider files. Before running this command you should have prepared profile (run sf prepare:profile) and provider (run sf prepare:provider)';
```

**Review by kysely — COMMENTED** [review]
Looks good.

I'm just thinking:
— `create:mock-map` could be simply `create:mock`?
— `create:mock-map-test` could be simply `create:mock-test`?

Or would it be too much to merge both into `create:mock`?

**Comment by Jakub-Vacek** [comment]
> I'm just thinking: — `create:mock-map` could be simply `create:mock`?
> `create:mock-map-test` could be simply `create:mock-test`?

Yep, I just wanted command name sto be self explanatory 🤷🏻‍♂️

> Or would it be too much to merge both into `create:mock`?
It depends on expected user flow. If we are expecting user who creates mock map to also need test for the we can do it :)

**Comment by kysely** [comment]
> Yep, I just wanted command name sto be self explanatory 🤷🏻‍♂️

Then maybe can we do alises?

**Review by kysely — APPROVED** [review]
I think it's good to go but first please go through docs and all places we used `create` command at and make sure the instructions are updated :)

**Comment by Jakub-Vacek** [comment]
> I think it's good to go but first please go through docs and all places we used `create` command at and make sure the instructions are updated :)

Yes, already on it :)

**Pull request merged**